capstone-frontend/src/ClubGridItem.test.js

```diff
@@ -15,24 +15,21 @@ document.body.appendChild(container);
 const club = {
   name: "name",
   description: "description",
-  ownerID: "ownerID",
-  gbookID: "gbookID"
+  gbook: "gbook", 
+  memberIDs: ["one", "two", "three"]
 };
 
 test('renders club tile test', () => {
-  render(<Router> <ClubGridItem name={club.name} description={club.description} ownerID={club.ownerID} gbookID={club.gbookID} /> </Router>, container);
+  render(<Router> <ClubGridItem club={club} /> </Router>, container);
```

Comment:
Nice job collapsing down these parameters into just the `club`!

Reply:
Thanks!